exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -413,4 +413,8 @@
 
   String DYNAMIC_UDF_SUPPORT_ENABLED = "exec.udf.enable_dynamic_support";
   BooleanValidator DYNAMIC_UDF_SUPPORT_ENABLED_VALIDATOR = new BooleanValidator(DYNAMIC_UDF_SUPPORT_ENABLED, true, true);
+
+  String USE_DYNAMIC_UDFS = "exec.udf.use_dynamic";
```

Comment:
Currently, we have one FunctionRegistryHolder (LocalFunctionRegistry) for both static and dynamic functions, from which we register for each query.  It gets updated when we download new jars/functions from zookeeper and will not be same as what it is during startup if dynamic UDF support is enabled and disabled.  That means if I use table I built during startup, it will miss the dynamic UDFs that got added in between and to include them, you have to rebuild the table.  For that reason, I choose to  add a new option.